### PR TITLE
check if the directory exists

### PR DESCRIPTION
**Pull request by elahehrashedi**
Some of the `checkFileExists` are replaced with `checkDirectoryExists`.
This will solve the issue for syncing between the configuration content in JSON and UI

### CODE REVIEW

**Review by michelleangela — APPROVED** [review]

File: Extension/src/LanguageServer/extension.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ export async function activate(activationEventOccurred: boolean): Promise<void>
     if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
         for (let i: number = 0; i < vscode.workspace.workspaceFolders.length; ++i) {
             const config: string = path.join(vscode.workspace.workspaceFolders[i].uri.fsPath, ".vscode/c_cpp_properties.json");
-            if (await util.checkFileExists(config)) {
+            if (await util.checkDirectoryExists(config)) {
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/extension.ts]: Isn't this a file and not a directory?

File: Extension/src/packageManager.ts [hunk 2]
```diff
@@ -384,7 +384,7 @@ export class PackageManager {
                             zipfile.readEntry();
                         });
                     } else {
-                        util.checkFileExists(absoluteEntryPath).then((exists: boolean) => {
+                        util.checkDirectoryExists(absoluteEntryPath).then((exists: boolean) => {
```
Comment (sean-mcmanus) [inline Extension/src/packageManager.ts]: Isn't this a file and not a directory? The "if" part above appears to handle directories.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/src/packageManager.ts (hunk 2 above)
Comment (elahehrashedi) [reply Extension/src/packageManager.ts]: I am not sure about this one.

File: Extension/src/LanguageServer/extension.ts (hunk 1 above)
Comment (elahehrashedi) [reply Extension/src/LanguageServer/extension.ts]: correct.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**